msData/attributeGroup/attgD003 test fails due to more than "200" attributes

The failure says: `JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".`

SE version:
```
lukas@m4 docker % java --version
openjdk 25.0.2 2026-01-20
OpenJDK Runtime Environment (build 25.0.2+10-69)
OpenJDK 64-Bit Server VM (build 25.0.2+10-69, mixed mode, sharing)
```

<details>
<summary>tests/xml_schema/msData/attributeGroup/jaxb/attgD003.xsd.html</summary>

```
 #Test Results (version 2)
#Wed Apr 29 15:25:08 CEST 2026
#-----testdescription-----
$file=/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/jaxb/attgD003.xsd.html
$root=/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests
Item=Value
executeArgs=-TestURL $testURL -xml ../attgD003.xml -package $package
executeClass=javasoft.sqe.tests.api.jakarta.xml.bind.JAXBValidationCheckerTest
id=attgD003.xml
keywords=document positive validation_checker
name=attgD003.xml
package=javasoft.sqe.tests.attributegroup.attgd003
schemaName=../attgD003.xsd
selectIf=toTestOptionalValidation
source=../attgD003.xsd
title=attgD003.xml - attgD003.xsd

#-----environment-----
command.execute=javasoft.sqe.javatest.lib.ExecStdTestOtherJVMCmd $xjc_env_vars CLASSPATH\=$testSuiteRootDir$/classes$\:${jaxbClasses}$\:$testSuiteRootDir$/lib$/javatest.jar$\:$testSuiteRootDir$/lib$/jtlegacy.jar$\:$testClassDir$\:$outDir $java_path -Djava.security.policy\=$testSuiteRootDir$/lib$/tck.policy $java_other_opt $testExecuteClass $testExecuteArgs
java_other_opt=
java_path=/Users/lukas/.sdkman/candidates/java/current/bin/java
jaxbClasses=/Users/lukas/development/github/jakartaee/jaxb-tck/checker.jar\:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jakarta.xml.bind-api.jar\:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jaxb-impl.jar\:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jaxb-jxc.jar\:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jakarta.activation-api.jar
xjc_default_operation_mode=true
xjc_sameOther=0

#-----testresult-----
description=file\:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/jaxb/attgD003.xsd.html\#attgD003.xml
end=Wed Apr 29 15\:25\:08 CEST 2026
environment=multi_jvm
execStatus=Failed. exit code 1
harnessLoaderMode=Classpath Loader
harnessVariety=Full Bundle
javatestOS=Mac OS X 26.4.1 (aarch64)
javatestVersion=6.0-ea+b24-2023-03-27-3294253
script=com.sun.jaxb_tck.lib.JaxbTckScript
sections=script_messages execute
start=Wed Apr 29 15\:25\:08 CEST 2026
test=xml_schema/msData/attributeGroup/jaxb/attgD003.xsd.html\#attgD003.xml
timeoutSeconds=600
totalTime=60
work=/Users/lukas/development/github/jakartaee/jaxb-tck/batch-multiJVM/work/xml_schema/msData/attributeGroup/jaxb

#section:script_messages
----------messages:(2/153)----------
Using compiled schema(s):
/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/jaxb/../attgD003.xsd

#section:execute
----------messages:(1/1365)----------
command: javasoft.sqe.javatest.lib.ExecStdTestOtherJVMCmd CLASSPATH=/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/classes:/Users/lukas/development/github/jakartaee/jaxb-tck/checker.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jakarta.xml.bind-api.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jaxb-impl.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jaxb-jxc.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jakarta.activation-api.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/lib/javatest.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/lib/jtlegacy.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/batch-multiJVM/work/classes:/Users/lukas/development/github/jakartaee/jaxb-tck/batch-multiJVM/work/classes/attributegroup/attgd003 /Users/lukas/.sdkman/candidates/java/current/bin/java -Djava.security.policy=/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/lib/tck.policy javasoft.sqe.tests.api.jakarta.xml.bind.JAXBValidationCheckerTest -TestURL file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/jaxb/attgD003.xsd.html#attgD003.xml -xml ../attgD003.xml -package javasoft.sqe.tests.attributegroup.attgd003 -schema ../attgD003.xsd
----------out1:(10/1221)----------
[Fatal Error] attgD003.xml:3:2512: JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".
Exception in thread "main" javasoft.sqe.jck.lib.MultiTestExt$Argument$InvalidValue: org.xml.sax.SAXParseException; systemId: file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/jaxb/../attgD003.xml; lineNumber: 3; columnNumber: 2512; JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".
	at javasoft.sqe.tests.api.jakarta.xml.bind.SchemaGenTestExt$XMLDocumentParameterValue.parseValue(SchemaGenTestExt.java:123)
	at javasoft.sqe.jck.lib.MultiTestExt.processParameter(MultiTestExt.java:313)
	at javasoft.sqe.jck.lib.MultiTestExt.decodeArg(MultiTestExt.java:288)
	at javasoft.sqe.javatest.lib.MultiTest.decodeAllArgs(MultiTest.java:259)
	at javasoft.sqe.javatest.lib.MultiTest.init(MultiTest.java:535)
	at javasoft.sqe.javatest.lib.MultiTest.run(MultiTest.java:178)
	at javasoft.sqe.javatest.lib.MultiTest.run(MultiTest.java:137)
	at javasoft.sqe.tests.api.jakarta.xml.bind.JAXBValidationCheckerTest.main(JAXBValidationCheckerTest.java:98)
----------out2:(0/0)----------
result: Failed. exit code 1


test result: Failed. exit code 1

```
</details>

<details>
<summary>tests/xml_schema/msData/attributeGroup/jaxb/attgD003.html</summary>

```
#Test Results (version 2)
#Wed Apr 29 15:25:08 CEST 2026
#-----testdescription-----
$file=/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/jaxb/attgD003.html
$root=/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests
Item=Value
document=../attgD003.xml
executeArgs=-TestURL $testURL -package $package -out \:ref -document ../attgD003.xml -pcontent attgd003-attgD003.jbx
executeClass=javasoft.sqe.tests.api.jakarta.xml.bind.JAXBTest
id=attgD003.v
jb-context=attgd003-attgD003.jbx
keywords=document positive
name=attgD003.v
package=javasoft.sqe.tests.attributegroup.attgd003
schemaName=../attgD003.xsd
source=../attgD003.xsd
title=attgD003.v - attgD003

#-----environment-----
command.execute=javasoft.sqe.javatest.lib.ExecStdTestOtherJVMCmd $xjc_env_vars CLASSPATH\=$testSuiteRootDir$/classes$\:${jaxbClasses}$\:$testSuiteRootDir$/lib$/javatest.jar$\:$testSuiteRootDir$/lib$/jtlegacy.jar$\:$testClassDir$\:$outDir $java_path -Djava.security.policy\=$testSuiteRootDir$/lib$/tck.policy $java_other_opt $testExecuteClass $testExecuteArgs
java_other_opt=
java_path=/Users/lukas/.sdkman/candidates/java/current/bin/java
jaxbClasses=/Users/lukas/development/github/jakartaee/jaxb-tck/checker.jar\:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jakarta.xml.bind-api.jar\:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jaxb-impl.jar\:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jaxb-jxc.jar\:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jakarta.activation-api.jar
xjc_default_operation_mode=true
xjc_sameOther=0

#-----testresult-----
description=file\:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/jaxb/attgD003.html\#attgD003.v
end=Wed Apr 29 15\:25\:08 CEST 2026
environment=multi_jvm
execStatus=Failed. test cases\: 3; all failed; first test case failure\: unmarshal
harnessLoaderMode=Classpath Loader
harnessVariety=Full Bundle
javatestOS=Mac OS X 26.4.1 (aarch64)
javatestVersion=6.0-ea+b24-2023-03-27-3294253
script=com.sun.jaxb_tck.lib.JaxbTckScript
sections=script_messages execute
start=Wed Apr 29 15\:25\:07 CEST 2026
test=xml_schema/msData/attributeGroup/jaxb/attgD003.html\#attgD003.v
timeoutSeconds=600
totalTime=186
work=/Users/lukas/development/github/jakartaee/jaxb-tck/batch-multiJVM/work/xml_schema/msData/attributeGroup/jaxb

#section:script_messages
----------messages:(2/153)----------
Using compiled schema(s):
/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/jaxb/../attgD003.xsd

#section:execute
----------messages:(1/1389)----------
command: javasoft.sqe.javatest.lib.ExecStdTestOtherJVMCmd CLASSPATH=/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/classes:/Users/lukas/development/github/jakartaee/jaxb-tck/checker.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jakarta.xml.bind-api.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jaxb-impl.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jaxb-jxc.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/jaxb-ri/mod/jakarta.activation-api.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/lib/javatest.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/lib/jtlegacy.jar:/Users/lukas/development/github/jakartaee/jaxb-tck/batch-multiJVM/work/classes:/Users/lukas/development/github/jakartaee/jaxb-tck/batch-multiJVM/work/classes/attributegroup/attgd003 /Users/lukas/.sdkman/candidates/java/current/bin/java -Djava.security.policy=/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/lib/tck.policy javasoft.sqe.tests.api.jakarta.xml.bind.JAXBTest -TestURL file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/jaxb/attgD003.html#attgD003.v -package javasoft.sqe.tests.attributegroup.attgd003 -out :ref -document ../attgD003.xml -pcontent attgd003-attgD003.jbx -schema ../attgD003.xsd
----------out1:(4/1086)----------
unmarshal: Failed. FATAL_ERROR at (3, 2512): JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".
marshal: Failed. jakarta.xml.bind.UnmarshalException  - with linked exception: [org.xml.sax.SAXParseException; systemId: file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/attgD003.xml; lineNumber: 3; columnNumber: 2512; JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".]
compareContent: Failed. The content comparison failed: jakarta.xml.bind.UnmarshalException  - with linked exception: [org.xml.sax.SAXParseException; systemId: file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/attgD003.xml; lineNumber: 3; columnNumber: 2512; JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".]
STATUS:Failed.test cases: 3; all failed; first test case failure: unmarshal
----------out2:(107/11834)----------
jakarta.xml.bind.UnmarshalException
 - with linked exception:
[org.xml.sax.SAXParseException; systemId: file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/attgD003.xml; lineNumber: 3; columnNumber: 2512; JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".]
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.createUnmarshalException(AbstractUnmarshallerImpl.java:296)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.createUnmarshalException(UnmarshallerImpl.java:546)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:233)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal(UnmarshallerImpl.java:198)
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:152)
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:157)
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:167)
	at javasoft.sqe.tests.api.jakarta.xml.bind.JAXBTest.unmarshal(JAXBTest.java:201)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at javasoft.sqe.javatest.lib.MultiTest.invokeTestCase(MultiTest.java:409)
	at javasoft.sqe.jck.lib.MultiTestExt.invokeTestCase(MultiTestExt.java:247)
	at javasoft.sqe.javatest.lib.MultiTest.run(MultiTest.java:205)
	at javasoft.sqe.tests.api.jakarta.xml.bind.JAXBTest.main(JAXBTest.java:51)
Caused by: org.xml.sax.SAXParseException; systemId: file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/attgD003.xml; lineNumber: 3; columnNumber: 2512; JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:178)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:284)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:254)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2736)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:635)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:113)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:551)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:890)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:826)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:137)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1225)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:629)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:227)
	... 11 more
Events:
0: FATAL_ERROR at (3, 2512): JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".
jakarta.xml.bind.UnmarshalException
 - with linked exception:
[org.xml.sax.SAXParseException; systemId: file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/attgD003.xml; lineNumber: 3; columnNumber: 2512; JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".]
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.createUnmarshalException(AbstractUnmarshallerImpl.java:296)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.createUnmarshalException(UnmarshallerImpl.java:546)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:233)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal(UnmarshallerImpl.java:198)
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:152)
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:157)
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:167)
	at javasoft.sqe.tests.api.jakarta.xml.bind.JAXBTest.marshal(JAXBTest.java:245)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at javasoft.sqe.javatest.lib.MultiTest.invokeTestCase(MultiTest.java:409)
	at javasoft.sqe.jck.lib.MultiTestExt.invokeTestCase(MultiTestExt.java:247)
	at javasoft.sqe.javatest.lib.MultiTest.run(MultiTest.java:205)
	at javasoft.sqe.tests.api.jakarta.xml.bind.JAXBTest.main(JAXBTest.java:51)
Caused by: org.xml.sax.SAXParseException; systemId: file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/attgD003.xml; lineNumber: 3; columnNumber: 2512; JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:178)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:284)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:254)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2736)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:635)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:113)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:551)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:890)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:826)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:137)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1225)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:629)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:227)
	... 11 more
jakarta.xml.bind.UnmarshalException
 - with linked exception:
[org.xml.sax.SAXParseException; systemId: file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/attgD003.xml; lineNumber: 3; columnNumber: 2512; JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".]
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.createUnmarshalException(AbstractUnmarshallerImpl.java:296)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.createUnmarshalException(UnmarshallerImpl.java:546)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:233)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal(UnmarshallerImpl.java:198)
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:152)
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:157)
	at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:167)
	at javasoft.sqe.tests.api.jakarta.xml.bind.JAXBTest.compareContent(JAXBTest.java:277)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at javasoft.sqe.javatest.lib.MultiTest.invokeTestCase(MultiTest.java:409)
	at javasoft.sqe.jck.lib.MultiTestExt.invokeTestCase(MultiTestExt.java:247)
	at javasoft.sqe.javatest.lib.MultiTest.run(MultiTest.java:205)
	at javasoft.sqe.tests.api.jakarta.xml.bind.JAXBTest.main(JAXBTest.java:51)
Caused by: org.xml.sax.SAXParseException; systemId: file:/Users/lukas/development/github/jakartaee/jaxb-tck/xml-binding-tck/tests/xml_schema/msData/attributeGroup/attgD003.xml; lineNumber: 3; columnNumber: 2512; JAXP00010002:  Element "elem" has more than "200" attributes, "200" is the limit set by "jdk.xml.elementAttributeLimit".
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:178)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:284)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:254)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2736)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:635)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:113)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:551)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:890)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:826)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:137)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1225)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:629)
	at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:227)
	... 11 more
result: Failed. test cases: 3; all failed; first test case failure: unmarshal


test result: Failed. test cases: 3; all failed; first test case failure: unmarshal
```

</details>

--- a/docker/run_jaxbtck.sh
+++ b/docker/run_jaxbtck.sh
@@ -97,7 +97,7 @@ if [ -z "$GF_BUNDLE_URL" ]; then
   export GF_BUNDLE_URL=$DEFAULT_GF_BUNDLE_URL
 fi
 export TCK_ROOT=${WORKSPACE}
-export  JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8 -DnoSecurityManager=true"
+export JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8 -DnoSecurityManager=true -Djdk.xml.elementAttributeLimit=2000"
 
 #wget $WGET_PROPS $GF_BUNDLE_URL -O latest-glassfish.zip
 

